GliaWidgets/View/Chat/Message/Content/Text/ChatTextContentView.swift

```diff
@@ -101,12 +104,37 @@ extension ChatTextContentView: UITextViewDelegate {
         in characterRange: NSRange,
         interaction: UITextItemInteraction
     ) -> Bool {
-        if URL.scheme == "tel" || URL.scheme == "mailto" {
-            return true
+        if URL.scheme == "tel" {
+            openPhoneDialer(url: URL)
+        } else if URL.scheme == "mailto" {
+            openEmailClient(url: URL)
         } else {
             linkTapped?(URL)
-            return false
         }
+
+        return false
+    }
+
+    private func openPhoneDialer(url: URL) {
+        let phone = url.absoluteString.replacingOccurrences(of: "tel:", with: "")
+
+        if let callUrl = URL(string: "tel://\(phone)"), environment.uiApplication.canOpenURL(callUrl) {
+            environment.uiApplication.open(callUrl)
+        }
```

Comment:
I may be wrong, but iOS should handle properly URL with `tel:` and `tel://`. Since we previously checked the URL scheme we need to check if OS can open URL and open it, or did I missed something?
```swift
private func openPhoneDialer(url: URL) {
    guard environment.uiApplication.canOpenURL(url) else { return }
    environment.uiApplication.open(url)
}
```

Reply:
I think you are correct, I was just following all the sources online I could find. They seemed to do the slashes thing.
But regarding to old docs it seems it should work OOO: https://developer.apple.com/library/archive/featuredarticles/iPhoneURLScheme_Reference/PhoneLinks/PhoneLinks.html#//apple_ref/doc/uid/TP40007899-CH6-SW1